mqtt/mqtt-broker/src/session/offline.rs

```diff
@@ -35,6 +36,10 @@ impl OfflineSession {
         self.state.subscriptions()
     }
 
+    pub fn last_active(&self) -> &DateTime<Utc> {
```

Comment:
```suggestion
    pub fn last_active(&self) -> DateTime<Utc> {
```
since `DateTime<Tz>` implements Copy